backend-api/template.yaml

```diff
@@ -225,4 +270,77 @@ Resources:
               - Effect: Allow
                 Action:
                   - kms:Sign
-                Resource: !ImportValue platform-kms-signing-key-arn
\ No newline at end of file
+                Resource: !ImportValue platform-kms-signing-key-arn
+
+# TxMA Audit service
+  TxMASQSQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 604800
+      VisibilityTimeout: 60
+      KmsMasterKeyId: !Ref TxMAKMSEncryptionKey
+      RedrivePolicy:
+        deadLetterTargetArn: !GetAtt TxMASQSQueueDeadLetterQueue.Arn
+        maxReceiveCount: 5
+
+
+# TxMA account consumes this queue
+  TxMASQSQueuePolicy:
```

Comment:
I don't think we should be giving TxMA access to Dev/Build so I'll move this into a conditional

Reply:
updated